lib/taskana-core-test/src/test/java/acceptance/workbasket/get/GetWorkbasketAccTest.java

```diff
@@ -58,12 +105,10 @@ void should_ReturnWorkbasketWithId_When_IdIsValidAndUserHasPermissions() throws
   @TestTemplate
   void should_ReturnWorkbasketByKeyAndDomain_When_NoExplicitPermissionButUserHasAdministrativeRole()
       throws Exception {
+    Workbasket workbasket = workbasketService.getWorkbasket(defaultWorkbasketSummary.getId());
```

Comment:
Test name says "byKeyAndDomain", but you use id to get a Workbasket. Please get Workbasket by id or rename the test.